digdag-docs/src/scheduling_workflow.rst

```diff
@@ -61,7 +61,7 @@ cron>: ``CRON``                 Use cron format for complex scheduling      cron
         ======================= ========================= =========================
         hourly>: "32:32"        2019-02-24 14:00:00 +0900 2019-02-24 14:32:32 +0900
         daily>: "10:32:32"      2019-02-25 00:00:00 +0900 2019-02-25 10:32:32 +0900
-        weekly>: "2,10:32:32"   2019-02-26 00:00:00 +0900 2019-02-24 14:32:32 +0900
+        weekly>: "2,10:32:32"   2019-02-24 00:00:00 +0900 2019-02-26 10:32:32 +0900
```

Comment:
Isn't the first session time here `2019-02-26` instead of `2019-02-24`?

Reply:
Fixed!! Thanks